SETUP.md

```diff
@@ -34,6 +34,24 @@ pip install recommenders[examples,spark]
 #   c. Run the notebook.
 ```
 
+## Setup for developers
```

Comment:
Capitalize d in developers for consistency

Reply:
good catch